src/App.tsx

```diff
@@ -26,15 +26,17 @@ function App() {
 
   const isDisabled = !username;
   return (
-    <div>
-      <h1>Welcome to the online Book Repository</h1>
-      <form onSubmit={handleLogin}>
-        <div>
-          <label>Username:</label>  
-          <input required name="username" type="text" value={username} onChange={handleUsernameChange} />
-        </div>
-        <button name="submit" type="submit" disabled={isDisabled}>Login</button>
-      </form>
+    <div className="form-center">
```

Comment:
1. I think the popular naming convention for classes is now ["BEM"](https://getbem.com/), are you going to follow some convention?
2. The names of the classes sound either presentational (`form-center` should always be centred), or tied to the CSS rules (`flex-container` would use flexbox), so if you want to change it to something else in the future, you will have to change the class name, is it forward compatible enough for you?
3. It would be good to use CSS (OR SCSS) modules, which try to guarantee isolation and prevent style leaks
CSS/SCSS modules are usually utilised by naming the CSS/SCSS file .module.css and importing the classes from its default export

So -
```
import styles from './Book.module.scss';

<div className={styles.Book}>I am styled by the .Book selector in the module</div>
```

That does not mean you cannot have global (non-module) CSS/SCSS as well. You can combine both of them, but you should only import the global CSS/SCSS from App.tsx - other files should use modules exclusively